cli/uninstall/uninstall.go

```diff
@@ -113,11 +113,25 @@ func UninstallProgram(program string, programVersion string, binDst string, head
 		}
 	}
 
+	versionsToDelete := []string{programVersion}
+	if program == search.ProgramTt {
+		versionsToDelete = getAllTtVersionFormats(programVersion)
+	}
+
 	var isSymlinkRemoved bool
-	isSymlinkRemoved, err = remove(program, programVersion, binDst, cmdCtx)
+	for _, verToDel := range versionsToDelete {
+		isSymlinkRemoved, err = remove(program, verToDel, binDst, cmdCtx)
+		if err != nil && err.Error() != util.ErrNoTtInstalled.Error() {
```

Comment:
```suggestion
		if err != nil && !errors.Is(err, ErrNoTtInstalled) {
```

Reply:
`errors.Is` is not working for this case. Fixed to second suggestion.

Reply:
> `errors.Is` is not working for this case

It seems like a problem becase `remove(` does not return the error variable as is. So it could be changed independently. It is better to return ErrNoTtInstalled from the `remove(` call (or from a nested call).

Reply:
It seems like a problem because the `remove(` call does not return the error variable as is. This is not obvious and makes the code difficult to maintain. It is better to return `ErrNoTtInstalled` from the `remove(` call (or from a nested call).

Reply:
Now `remove(` is returning defined `errNoTtInstalled` when program name is "tt". It is now possible to use `errors.Is`.